React/Views/RCTView.m

```diff
@@ -118,6 +118,11 @@ - (instancetype)initWithFrame:(CGRect)frame
     _hitTestEdgeInsets = UIEdgeInsetsZero;
 
     _backgroundColor = super.backgroundColor;
+
+    [[NSNotificationCenter defaultCenter] addObserver:self
+                                             selector:@selector(voiceOverStatusDidChange:)
+                                                 name:UIAccessibilityVoiceOverStatusChanged
+                                               object:nil];
```

Comment:
i'm a little bit concerned about adding a new observer for *every* RCTView instantiated. That could be thousands of views. Can we figure out a way to do this with only a single observer?